typeset/converter.js

```diff
@@ -0,0 +1,189 @@
+const path = require('path')
+const fs = require('fs')
+const pify = require('pify')
+const puppeteer = require('puppeteer')
+const writeFile = pify(fs.writeFile)
+require('dotenv').config()
+const bunyan = require('bunyan')
+let level = process.env.LOG_LEVEL || 'info'
+let log = bunyan.createLogger({
+  name: 'typeset converter',
+  streams: [
+    {
+      level: level,
+      stream: process.stdout
+    },
+    {
+      type: 'rotating-file',
+      path: path.join(__dirname, `/logs/log-typeset-converter-${Date.now()}.log`),
+      period: '1d',
+      count: 30
+    }
+  ]
+})
+
+const injectMathJax = async (inputPath, cssPath, outputPath, dirname) => {
+  function clearTerminal () {
+    if (process.platform === 'darwin') {
+      process.stdout.write('\x1Bc')
```

Comment:
@PiotrKozlowski Is there a need to clear the terminal when running this conversion? I have found it useful to scroll up to see previous commands and output.